client/analytics/ad-tracking.js

```diff
@@ -181,6 +210,12 @@ module.exports = {
 		nextFunction();
 	},
 
+	recordAddToCart: function( cartItem ) {
```

Comment:
Why have a separate function, rather than just put the config check in the _other_ `recordAddToCart`? (I am sure there's a good reason :) )